sensor/common/detector/detector.go

```diff
@@ -474,10 +488,28 @@ func (d *detectorImpl) ProcessDeployment(ctx context.Context, deployment *storag
 		return
 	default:
 	}
+	d.deploymentsQueue.Push(&queue.DeploymentQueueItem{
+		Ctx:        ctx,
+		Deployment: deployment,
+		Action:     action,
+	})
```

Comment:
This is style only: don't you want to place this block inside the `default` branch of the `select`?

Reply:
Done